api/server/server.go

```diff
@@ -258,33 +223,32 @@ func (s *Server) bindHandlers(ctx context.Context) {
 	engine.GET("/stats", s.handleStats)
 
 	v1 := engine.Group("/v1")
-	v1.Use(s.middlewareWrapperFunc(ctx))
```

Comment:
Oh, I think we can revert it to middleware.
I was using wrapper because I using it to the background context to the request, but since we are just using *gin.Context, is not needed anymore.
I'll revert it